utils/azure_users.py

```diff
@@ -263,7 +263,8 @@ def get_groups_and_users(self):
             [member['objectId'] for member in item['members']],
         )
         result = list(map(parse_item, response.json()['value']))
-        result[0][1].append(self.config.USERID)
+        list_of_userid = self.config.USERID.split(" ")
```

Comment:
- change ** list_of_userid ** to ** users_id ** 
- I recommend using `,` as the split symbol.

Reply:
Resolve comment